cli/azd/pkg/tools/bicep/bicep.go

```diff
@@ -92,34 +79,31 @@ func (cli *bicepCli) CheckInstalled(ctx context.Context) (bool, error) {
 	return true, nil
 }
 
-func (cli *bicepCli) Build(ctx context.Context, file string) (string, error) {
-	sniffCliVersion := func() (string, error) {
-		verRes, err := cli.runCommand(ctx, "version", "--out", "json")
-		if err != nil {
-			return "", fmt.Errorf("failing running az version: %s (%w)", verRes.String(), err)
-		}
+func getBicepCliPath() (string, error) {
+	if found, err := osexec.LookPath("bicep"); err == nil {
+		return found, nil
+	}
 
-		var jsonVer struct {
-			AzureCli string `json:"azure-cli"`
-		}
+	usr, err := user.Current()
+	if err != nil {
+		return "", osexec.ErrNotFound
+	}
 
-		if err := json.Unmarshal([]byte(verRes.Stdout), &jsonVer); err != nil {
-			return "", fmt.Errorf("parsing cli version json: %s: %w", verRes.Stdout, err)
-		}
+	cmdName := "bicep"
+	if runtime.GOOS == "windows" {
```

Comment:
Not for this PR, and not a blocker at all,  but, I would propose to use the go pattern of naming files as `.linux` or `.windows` and create a PAL (or many PALs) instead of depending on `runtime`.  It is usually better to depend on compile-time phase than runtime, IMO

Reply:
I moved to this model.